### DISCUSSION TITLE
Unable to hide others repository (which is not related to me) from top repository on homepage.

### DISCUSSION THREAD

**Discussion by vishal273007** [community/community#145053] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I asked ai assistant, tried and did experiment but I'm unable to remove or hide stubborn repository that is unwilling showing on my GitHub homepage in top repository section.
![Screenshot_2024-11-19-22-04-23-090_com miui home](https://github.com/user-attachments/assets/314b53eb-fb94-4709-a841-32e777600cf8)

**Comment by shinybrightstar** [comment]
Hi @vishal273007 ,

The Feed team shipped an update to address this issue on November 5, 2024. It sounds like you are still experiencing issues with this. Please comment directly on [this discussion](https://github.com/orgs/community/discussions/143676) with any questions or additional details about your issue, and we will provide updates in that thread when possible.

We will be closing this discussion as a duplicate.

Thank you for your patience as we get Feeds back to normal for all users!
